app/assets/styles/styles.css

```diff
@@ -18,6 +18,12 @@ body {
   font-family: 'Roboto', sans-serif;
 }
 
+nav {
+  font-family: 'Trajan Pro Bold', 'Times New Roman', serif;
+  background: #1a1451;
+  padding: 0.5em 1.5em;
+}
```

Comment:
We want to avoid applying styles to elements unless they are resets or styles that we consciously want to use on all elements. 

This will prevent us from using a `nav` element on the site without have to override. Can we apply these styles to a class?